www/content/ci.md

```diff
@@ -266,3 +266,56 @@ data:
 
 Check [Managing Secrets](https://docs.semaphoreci.com/article/51-secrets-yaml-reference) for
 more detailed documentation.
+
+## GitLab CI
+
+To push releases to both GitHub and the **official** Docker registry, add a file `.gitlab-ci.yml` in the Go project directory:
+
+```yaml
+image: docker:stable
+services:
+- docker:dind
+
+stages:
+- build
+
+variables:
+  GORELEASER_IMAGE: goreleaser/goreleaser:v0.107.0
```

Comment:
maybe change to `latest`?

Reply:
Fixed.
Also remove the empty `before_script:`.